projects/gateway/pkg/services/k8sadmisssion/validating_admission_webhook_test.go

```diff
@@ -146,6 +149,48 @@ var _ = Describe("ValidatingAdmissionWebhook", func() {
 
 		})
 	})
+
+	Context("namespace scoping", func() {
+		It("Does not process the resource if it's not whitelisted by watchNamespaces", func() {
+			wh.alwaysAccept = false
+			wh.watchNamespaces = []string{routeTable.Metadata.Namespace}
+			wh.webhookNamespace = routeTable.Metadata.Namespace
+
+			req, err := makeReviewRequestRaw(srv.URL, v1.RouteTableCrd.GroupVersionKind(), v1beta1.Create, routeTable.Metadata.Name, routeTable.Metadata.Namespace+"other", []byte(`{"metadata": [1, 2, 3]}`))
+			Expect(err).NotTo(HaveOccurred())
+
+			res, err := srv.Client().Do(req)
+			Expect(err).NotTo(HaveOccurred())
+
+			review, err := parseReviewResponse(res)
+			Expect(err).NotTo(HaveOccurred())
+			Expect(review.Response).NotTo(BeNil())
+
+			Expect(review.Response.Allowed).To(BeTrue())
+			Expect(review.Response.Result).To(BeNil())
+		})
+
+		It("Does not process other-namespace gateway resources if readGatewaysFromAllNamespaces is false, even if they're from whitelisted namespaces", func() {
```

Comment:
super nit, typically we've been doing lower case after `It(`

```suggestion
		It("does not process other-namespace gateway resources if readGatewaysFromAllNamespaces is false, even if they're from whitelisted namespaces", func() {
```